### PR TITLE
add all platform for all robots

### PR DESCRIPTION
**Pull request by knorth55**
add `all` for platform

### CODE REVIEW

**Comment by v4hn** [comment]
From the patch it looks like *not specifying* a platform at all might do the same as the proposed "all" tag. Do we really need that patch?

**Comment by knorth55** [comment]
not specifying platform is also for all robots.
but i think it is good to have `platform: all` too for readability of config file of apps.

File: src/app_manager/app_list.py [hunk 1]
```diff
@@ -129,7 +129,7 @@ def update(self):
             self._file_mtime = s.st_mtime
 
     def get_available_apps(self, platform=None):
-        if platform is not None:
+        if (platform is not None or platform != 'all'):
```
Comment (v4hn) [inline src/app_manager/app_list.py]: This should be `and` instead of `or`, right?

**Review by v4hn — COMMENTED** [review]
I'm ok with the idea to add `all` as a keyword.
Please look at the inline comment though.

I will leave the decision to merge this to @k-okada as the patch comes from your lab.

File: src/app_manager/app_list.py (hunk 1 above)
Comment (knorth55) [reply src/app_manager/app_list.py]: yes, thats true, thank you!

**Review by v4hn — APPROVED** [review]

**Comment by v4hn** [comment]
Thanks for all your minor fixes @knorth55 ! :+1:

**Pull request merged**